workflow/studio.mdx

```diff
@@ -38,3 +39,22 @@ npx novu@latest dev --port <YOUR_SERVER_PORT> --route <YOUR_NOVU_ROUTE_PATH>
 
 - **YOUR_SERVER_PORT** - This accepts the port number where your server is running. Defaults to 4000.
 - **YOUR_NOVU_ROUTE_PATH** - This is the mounted path of the framework `serve` function. Defaults to `/api/novu`.
+
+## Novu CLI flags
+
+The Novu CLI command `npx novu@latest dev` supports a number of flags:
+
+| flag | long form usage example   | description                   | default value               |
+| ---- | -----------------------   | ----------------------------- | --------------------------- |
+| -p   | --port `<port>`           | local bridge application port | 4000                        |
+| -r   | --route `<route>`         | bridge application route      | /api/novu                   |
+| -o   | --origin `<origin>`       | bridge application origin     | http://localhost            |
+| -d   | --dashboard-url `<url>`   | Novu cloud dashboard URL      | https://dashboard.novu.co   |
+| -sp  | --studio-port `<port>`    | Local Studio server port      | 2022                        |
+| -t   | --tunnel `<url>`          | self hosted tunnel url        | https://my-tunnel.ngrok.app |
```

Comment:
Using uppercase consistently, and fixing fixing the tunnel default (it doesn't have one). Also, it's not a self-hosted tunnel (other platforms host it), it's just a custom tunnel.

```suggestion
| Flag | Long form usage example   | Description                   | Default value               |
| ---- | -----------------------   | ----------------------------- | --------------------------- |
| -p   | --port `<port>`           | Local bridge application port | 4000                        |
| -r   | --route `<route>`         | Bridge application route      | /api/novu                   |
| -o   | --origin `<origin>`       | Bridge application origin     | http://localhost            |
| -d   | --dashboard-url `<url>`   | Novu cloud dashboard URL      | https://dashboard.novu.co   |
| -sp  | --studio-port `<port>`    | Local Studio server port      | 2022                        |
| -t   | --tunnel `<url>`          | Custom tunnel url        | null |
```